src/mcts/node.h

```diff
@@ -160,6 +161,11 @@ class Node {
   uint16_t GetMaxDepth() const { return max_depth_; }
   uint16_t GetNumEdges() const { return edges_.size(); }
 
+  // Calculates score in centipawns.
+  float GetCpScore() const {
```

Comment:
Should just return an `int` I think, to match the `ThinkingInfo` type, which is typed as such because the precision is specifically capped at .01 pawns